totalRP3/Modules/Register/Characters/RegisterCharacteristics.lua

```diff
@@ -882,51 +882,57 @@ local function miscInfoFixupPosition(frame, relative)
 	-- and preserve all the other attributes.
 	for i = 1, frame:GetNumPoints() do
 		local point, _, relPoint, x, y = frame:GetPoint(i);
-		if point == "TOP" and relPoint == "BOTTOM" then
+		if point:find("^TOP") and relPoint:find("^BOTTOM") then
 			frame:SetPoint(point, relative, relPoint, x, y);
 			return;
 		end
 	end
 end
 
---- miscInfoPerformReorder reorders the character frame info list, moving
---  the item at a given source index to that of a target index, and updating
---  the layout of the UI.
---
---  @param sourceIndex The index of the node being moved.
---  @param targetIndex The index to place the node at.
-local function miscInfoPerformReorder(sourceIndex, targetIndex)
+--- reorderInfoPerformReorder reorders the character frame info list, moving
+--- the item at a given source index to that of a target index, and updating
+--- the layout of the UI.
+---@param data table The info data it requires: draftData.MI or draftData.PS.
+---@param editCharFrame Frame miscEditCharFrame or psychoEditCharFrame to check.
+---@param sourceIndex number The index of the node being moved.
+---@param targetIndex number The index to place the node at.
+---@param editTitle Frame The character edit title.
+---@param addBtn Button The add more button.
+local function reorderInfoPerformReorder(data, editCharFrame, sourceIndex, targetIndex, editTitle, addBtn, choice)
 	-- We'll just do a flat table order change here with a tinsert/tremove.
-	local source = table.remove(draftData.MI, sourceIndex);
-	table.insert(draftData.MI, targetIndex, source);
+	local source = table.remove(data, sourceIndex);
+	table.insert(data, targetIndex, source);
 
 	-- Reorder the frame too. Means we don't need to transfer all the values,
 	-- which fixes issues with icons not persisting correctly when reordering.
-	local sourceFrame = table.remove(miscEditCharFrame, sourceIndex);
-	table.insert(miscEditCharFrame, targetIndex, sourceFrame);
+	local sourceFrame = table.remove(editCharFrame, sourceIndex);
+	table.insert(editCharFrame, targetIndex, sourceFrame);
 
 	-- Now fix up all the frames in terms of their referenced indices.
-	local previous = TRP3_RegisterCharact_CharactPanel_Edit_MiscTitle;
-	for frameIndex, _ in pairs(draftData.MI) do
-		local frame = miscEditCharFrame[frameIndex];
-		frame.miscIndex = frameIndex;
+	local previous = editTitle;
+	for frameIndex, _ in pairs(data) do
```

Comment:
```suggestion
	for frameIndex in ipairs(data) do
```
This was technically always wrong but.,..